File: ComputingCP/Final/CoinsofComputingRepublic/Solver.cpp
#include <bits/stdc++.h>
using namespace std;
typedef int long il;

int main() {
  il n2 = 0, n5 = 0, n;
  cin >> n;
  if (n == 1 || n == 3) {
    cout << "impossible\n";
  } else if (n == 5) {
    cout << "0 1\n";
  } else {
    if(n%5 == 0) {
        n5 = n/5;
    } else {
      do {
        n = n - 2;
        n2++;
      } while (n % 5 != 0);
      n5 = n / 5;
    }
    cout << n2 << " " << n5 << endl;
  }
  return 0;
}

File: ComputingCP/WarmingUp/FindCertainNumber/Solver.java
#include <bits/stdc++.h>
using namespace std;

int main() {
      int n,x;cin >> n >> x;
        string str = "NOT FOUND!";
	for(int i = 0;i < n;i++) {
	        int a;cin >> a;
		    if(a == x) {
			str = "FOUND IT!";
			break;
		}	      
	}
        cout << str << endl;
	    return 0;

}
